src/lib/config.js

```diff
@@ -6,7 +6,8 @@ module.exports = {
     PATHS: {
       SESSION: "session",
       CHECK: "check-driving-licence",
-      AUTHORIZATION: "authorization"
+      AUTHORIZATION: "authorization",
+      PERSON_INFO: "person-info"
```

Comment:
Is this correct its different to whats on the ticket

Reply:
Endpoint on ticket was taken from a similar piece of work kiwi did so an internal decision not linked to any requirement, I opted to add the hyphen to keep inline with existing convention